src/FizzWare.NBuilder/Generators/Usa.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FizzWare.NBuilder.Generators
+{
+    /// <summary>
+    /// This class can randomly generate data specifically for the United States.
+    /// </summary>
+    public static class Usa

```

Comment:
If obsoleteing the States and Cities, etc ... why not the entire class?

Reply:
I'm planning to keep `Usa` around--but as a separate thing from `GetRandom`.  This is the new class. The original one in `GetRandom` should be obsoleted.